Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.Android/MapRenderer.cs

```diff
@@ -27,14 +28,14 @@ public class MapRenderer : ViewRenderer<Map, global::Android.Views.View>,
     {
         readonly CameraLogic _cameraLogic;
         readonly UiSettingsLogic _uiSettingsLogic = new UiSettingsLogic();
-        readonly BaseLogic<GoogleMap>[] _logics;
+        internal readonly List<BaseLogic<GoogleMap>> _logics;
```

Comment:
Could you change to ``IList<BaseLogic<GoogleMap>>`` ?

Reply:
Yes, sure. I also renamed the field to match a naming convention (since it's no longer a private field)